x-pack/plugins/secops/public/components/timeline/data_providers/facet_text.tsx

```diff
@@ -16,18 +17,7 @@ interface Props {
  * typically a numeric value, but non-numeric values can also be rendered.
  */
 export const FacetText = pure<Props>(({ text }) => (
-  <span
-    data-test-subj="facetText"
-    style={{
-      backgroundColor: '#e4e4e4',
-      border: '1px solid',
-      borderColor: '#e4e4e4',
-      borderRadius: '4px',
-      color: '#666666',
-      margin: '0px 5px',
-      padding: '0 2px 0 2px',
-    }}
-  >
-    {text}
-  </span>
+  <EuiBadge color="primary">
```

Comment:
Perhaps we should use this? https://elastic.github.io/eui/#/navigation/facet

Reply:
Ok, new issue for it:
https://github.com/elastic/ingest-dev/issues/92